birdhouse/deployment/triggerdeploy.sh

```diff
@@ -119,6 +119,8 @@ START_TIME="`date -Isecond`"
 echo "==========
 triggerdeploy START_TIME=$START_TIME"
 
+. ./default.env
```

Comment:
`$COMPOSE_DIR/default.env`

Reply:
For testing, i used the daccs-iac, with the birdhouse stack, and I checked the log of the autodeploy cronjob and the job was running without errors. I planned to check also after the merge on a staging setup.